src/login_param.rs

```diff
@@ -44,13 +49,112 @@ pub struct ServerLoginParam {
     pub certificate_checks: CertificateChecks,
 }
 
+#[derive(Default, Debug, Clone, PartialEq)]
+pub struct DeltaSocks5Config {
```

Comment:
prefixing with Delta in the delta codebase seems strange to me

Reply:
I overhauled that struct, it's now called Socks5Config and is a simple wrapper around async_smtp::smtp::Socks5Config